### ISSUE TITLE
[Minor]: Table Misalignment in Planners: A to Z

### ISSUE THREAD

**Issue by MeFisto94** [nergmada/planning-wiki#88]
Link to page(s) with issue: 
https://planning.wiki/ref/planners/atoz

Quote of current error:
![grafik](https://user-images.githubusercontent.com/12577813/100993947-52f24200-3556-11eb-9dc9-95e1d4e41bd6.png)
The columns start to get misaligned and the links to the wiki pages are missing.

Proposed correction:
Fix the misalignment, which probably also fixes the linking

**Comment by Maumagnaguagno** [comment]
You are right, there is a problem in the actual [table data](https://github.com/nergmada/planning-wiki/blob/master/docs/reference/planners/atoz.md).
Is is very easy to fix, I will try to do it now.

Not sure about your edit, FD is part of the satisfycing table.
Some planners are just not part of the table as the wiki is under construction.

**Comment by MeFisto94** [comment]
Regarding the edit: Sorry, I didn't pay attention, it was about [FF](https://planning.wiki/ref/planners/ff) which has the satisfycing tag, but is not listed in the Satisfycing "overview".

It's not related because I thought it's related to the table data error, but apparently this happens after HSP (which is also the last entry on the by-tag page), so that one is unrelated, but I guess a few other satisfycing planners will appear on the by-tag page once this is merged?

**Comment by Maumagnaguagno** [comment]
Oh, I get it.
You expect that FF being tagged as satisfycing in its own page would automatically make it part of the satisfycing table.
Unfortunately the system is not that smart, link tags are added by hand.
You can see the actual table [here](https://github.com/nergmada/planning-wiki/blob/master/docs/reference/planners/tags/satisfycing.md) also lacking FF.
